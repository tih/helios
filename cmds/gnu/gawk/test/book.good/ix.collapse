[FS] variable	35 36 52
[if]-[else] statement	47
arithmetic operators	36 44
coercion rules	44
coercion~to number	45
coercion~to string	45
control-flow statements	48
numeric comparison	44
string comparison	44
