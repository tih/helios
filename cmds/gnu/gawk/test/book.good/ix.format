.XX
arithmetic operators  36, 44
.XX
coercion rules  44
.XX
coercion to number  45
.XX
coercion to string  45
.XX
comparison, numeric  44
.XX
comparison, string  44
.XX
control-flow statements  48
.XX
\f(CWFS\fP variable  35, 36, 52
.XX
\f(CWif\fP-\f(CWelse\fP statement  47
.XX
number, coercion to  45
.XX
numeric comparison  44
.XX
operators, arithmetic  36, 44
.XX
rules, coercion  44
.XX
statement, \f(CWif\fP-\f(CWelse\fP  47
.XX
statements, control-flow  48
.XX
string, coercion to  45
.XX
string comparison  44
.XX
variable, \f(CWFS\fP  35, 36, 52
