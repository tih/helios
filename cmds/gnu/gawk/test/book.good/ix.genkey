FS variable	[FS] variable	35 36 52
variable FS	variable, [FS]	35 36 52
ifelse statement	[if]-[else] statement	47
statement ifelse	statement, [if]-[else]	47
arithmetic operators	arithmetic operators	36 44
operators arithmetic	operators, arithmetic	36 44
coercion rules	coercion rules	44
rules coercion	rules, coercion	44
coercion to number	coercion to number	45
number coercion to	number, coercion to	45
coercion to string	coercion to string	45
string coercion to	string, coercion to	45
controlflow statements	control-flow statements	48
statements controlflow	statements, control-flow	48
numeric comparison	numeric comparison	44
comparison numeric	comparison, numeric	44
string comparison	string comparison	44
comparison string	comparison, string	44
