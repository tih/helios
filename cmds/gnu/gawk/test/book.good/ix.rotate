[FS] variable	35 36 52
variable, [FS]	35 36 52
[if]-[else] statement	47
statement, [if]-[else]	47
arithmetic operators	36 44
operators, arithmetic	36 44
coercion rules	44
rules, coercion	44
coercion~to number	45
number, coercion~to	45
coercion~to string	45
string, coercion~to	45
control-flow statements	48
statements, control-flow	48
numeric comparison	44
comparison, numeric	44
string comparison	44
comparison, string	44
