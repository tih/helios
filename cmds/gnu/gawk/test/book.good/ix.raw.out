[FS] variable	35
[FS] variable	36
[FS] variable	52
[if]-[else] statement	47
arithmetic operators	36
arithmetic operators	44
coercion rules	44
coercion~to number	45
coercion~to string	45
control-flow statements	48
numeric comparison	44
string comparison	44
